Mathlib/Topology/Sets/VietorisTopology.lean

```diff
@@ -0,0 +1,157 @@
+/-
+Copyright (c) 2025 Attila Gáspár. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Attila Gáspár
+-/
+module
+
+public import Mathlib.Topology.Sets.Compacts
+
+/-!
+# Vietoris topology
+
+This file defines the Vietoris topology on the types of compact subsets and nonempty compact subsets
+of a topological space.
```

Comment:
It would be nice if this module docstring could be more detailed, i.e give at least an informal description of what this topology is and provide a reference. Same for the actual definition `vietoris` below.